### ISSUE TITLE
can't install vscode

### ISSUE THREAD

**Issue by pseudocubic** [NixOS/nixpkgs#138729]
### Describe the bug
Vscode fails to build during `patching sources` step

`Error: ENOENT: no such file or directory, open 'resources/app/node_modules.asar'`

### Steps To Reproduce
1. `nix-shell -p vscode` or `nix-env -iA nixpkgs.vscode`

### Expected behavior
Vscode to install into environment

### Screenshots
```bash
installing 'vscode-1.60.1'
these 4 derivations will be built:
  /nix/store/5bp20nl1yyxsif42y2hzfizk7gsyvh1w-code.desktop.drv
  /nix/store/b1dj9z5qmwf4qkcdfb3mqaw6gqsp6wqn-VSCode_1.60.1_darwin.zip.drv
  /nix/store/ylnzdcy12kdarq9lxbby8bbk4f2fds4j-code-url-handler.desktop.drv
  /nix/store/i17ppaa3z36kzcknbmkrg552fda5zffs-vscode-1.60.1.drv
these 43 paths will be fetched (131.03 MiB download, 651.27 MiB unpacked):
  /nix/store/0mdzl24li08cq8qq8hbp8m1xh03zpa78-cctools-binutils-darwin-wrapper-949.0.1
  /nix/store/13ssn77ivl8bq4w6snd2d945vrdh3i2h-libtapi-1100.0.11
  /nix/store/17mkcc40sfzphy9722gy92l0rgnv4z9y-openssl-1.1.1l
  /nix/store/2dpd685mlhdpdvvnnhshjgbb5n4113cb-stdenv-darwin
  /nix/store/2i6iscy7pqkykqp904iplbc7zb2r6z2i-nghttp2-1.43.0-bin
  /nix/store/3v5asy4nb2yqkb3asgvpv6ya7nzh09yh-clang-wrapper-7.1.0
  /nix/store/3wvdw1mwa2wl2lbwy6ap30ymzk1ximbv-libsecret-0.20.4
  /nix/store/4l0ckp42ifs8srb80k70bzkh958p8mwi-libxshmfence-1.3
  /nix/store/4ybj3xpyr0w0h784fjmmz18w9q64y06g-unzip-6.0
  /nix/store/52nicn2hsjsaj9y2gv49gg72qc48jrfz-libsecret-0.20.4-dev
  /nix/store/8ndqi7c6mgwm95firnfmh3flv0hsrfxp-libXScrnSaver-1.2.3
  /nix/store/8s5mpkpqlssrkcz9ixkhspkwl23bwidw-binutils-2.35.1
  /nix/store/8vy1z8pbswdishqp62824dbxvgnx9z6z-xorgproto-2021.4
  /nix/store/8zgkak4qyd7ylrvw1zn27zq9ymnjdy8x-desktop-file-utils-0.24
  /nix/store/9kxlyddw3zgjsrvzxy8hb1nfa1jasgkp-openssl-1.1.1l-bin
  /nix/store/9mlkdg49j77i55v27jdl7kwcza5vvimk-stdenv-darwin
  /nix/store/ax3zwn40lc1cpkvizj1f2yc54rprs2k2-asar-3.0.3
  /nix/store/b5gcf12ij0485gqhbqbnkbisprkl6ij6-curl-7.76.1-bin
  /nix/store/c7z59k9yv0yyxlrrj1bar5pzg9356k5z-clang-7.1.0
  /nix/store/czyirh1hh3b9h106iarfj129qv00p3yv-libssh2-1.9.0-dev
  /nix/store/dw1dg92xgiix6xldxwkq5809c1jhld9v-cctools-port-949.0.1
  /nix/store/g0bfrpmfrq45c7fjqc6vg75g8zjg48hw-gnumake-4.3
  /nix/store/h5bsb60pqsizr070084v4448bvvgpk76-curl-7.76.1-dev
  /nix/store/h9vhrljwiw5hs5pv5y8524h85910vfr7-c-ares-1.17.2
  /nix/store/hdw7ingz4paq6jq4n341xgfj3xhf9ywv-clang-7.1.0-lib
  /nix/store/hh47i379mm4cbb2dws8gfw712kmdgn9z-nghttp2-1.43.0-lib
  /nix/store/kc9synymflz7qhxbbdrpqiw0lxwg6cpx-nghttp2-1.43.0-dev
  /nix/store/lp8kj8njk09vgqcl3qk5s3542d4njwym-libev-4.33
  /nix/store/n5zfdzjcdvhf2nkb5xzyqic5brnlcliq-adv_cmds-119-locale
  /nix/store/nn5l68h9m0dyfmd0wmxhnhgjfbwyrs9p-llvm-7.1.0-lib
  /nix/store/pbqxb6v7q1zg804hjif84s9nmh9adlmy-ed-1.17
  /nix/store/qh0bysjw5npwdyl6yppbcizlwp2339n5-patch-2.7.6
  /nix/store/ql5gnzmi02fr566g0wavvls84wffg1j6-llvm-7.1.0
  /nix/store/sicx2vnd45jk1fn1c14cpwi8qzzs8x54-libssh2-1.9.0
  /nix/store/swi24jhphnkwbwnwlknb7m147s4ix99x-cctools-binutils-darwin-949.0.1
  /nix/store/vqny9qpqdppmqw51v2jsc72pil3vsii0-nghttp2-1.43.0
  /nix/store/vzab3fdy9cvr3sg34mfw0rgbp4p4vj7k-curl-7.76.1
  /nix/store/wynzvldzsrgv4z4w1sxx8awh2rzdzh1s-libxshmfence-1.3-dev
  /nix/store/x2p8w4wmdxv6g2i320kljhiiz5idp4h1-gawk-5.1.0
  /nix/store/x5cp0jqsg9jz26n3a3vzl53b3wdy71ga-openssl-1.1.1l-dev
  /nix/store/xcd5wwbsyr38mvrlqfhfgml54fgpa6jh-mirrors-list
  /nix/store/y7s50z4ka86j5zjld625dfwpp8iw85ww-expand-response-params
  /nix/store/zla7029qmpxm2i80n1v1bc6lkg6mf3s0-curl-7.76.1-man
copying path '/nix/store/n5zfdzjcdvhf2nkb5xzyqic5brnlcliq-adv_cmds-119-locale' from 'https://cache.nixos.org'...
copying path '/nix/store/ax3zwn40lc1cpkvizj1f2yc54rprs2k2-asar-3.0.3' from 'https://cache.nixos.org'...
copying path '/nix/store/8s5mpkpqlssrkcz9ixkhspkwl23bwidw-binutils-2.35.1' from 'https://cache.nixos.org'...
copying path '/nix/store/h9vhrljwiw5hs5pv5y8524h85910vfr7-c-ares-1.17.2' from 'https://cache.nixos.org'...
copying path '/nix/store/zla7029qmpxm2i80n1v1bc6lkg6mf3s0-curl-7.76.1-man' from 'https://cache.nixos.org'...
copying path '/nix/store/8zgkak4qyd7ylrvw1zn27zq9ymnjdy8x-desktop-file-utils-0.24' from 'https://cache.nixos.org'...
copying path '/nix/store/pbqxb6v7q1zg804hjif84s9nmh9adlmy-ed-1.17' from 'https://cache.nixos.org'...
copying path '/nix/store/y7s50z4ka86j5zjld625dfwpp8iw85ww-expand-response-params' from 'https://cache.nixos.org'...
copying path '/nix/store/x2p8w4wmdxv6g2i320kljhiiz5idp4h1-gawk-5.1.0' from 'https://cache.nixos.org'...
copying path '/nix/store/g0bfrpmfrq45c7fjqc6vg75g8zjg48hw-gnumake-4.3' from 'https://cache.nixos.org'...
copying path '/nix/store/8ndqi7c6mgwm95firnfmh3flv0hsrfxp-libXScrnSaver-1.2.3' from 'https://cache.nixos.org'...
copying path '/nix/store/lp8kj8njk09vgqcl3qk5s3542d4njwym-libev-4.33' from 'https://cache.nixos.org'...
copying path '/nix/store/3wvdw1mwa2wl2lbwy6ap30ymzk1ximbv-libsecret-0.20.4' from 'https://cache.nixos.org'...
copying path '/nix/store/13ssn77ivl8bq4w6snd2d945vrdh3i2h-libtapi-1100.0.11' from 'https://cache.nixos.org'...
copying path '/nix/store/52nicn2hsjsaj9y2gv49gg72qc48jrfz-libsecret-0.20.4-dev' from 'https://cache.nixos.org'...
copying path '/nix/store/dw1dg92xgiix6xldxwkq5809c1jhld9v-cctools-port-949.0.1' from 'https://cache.nixos.org'...
copying path '/nix/store/4l0ckp42ifs8srb80k70bzkh958p8mwi-libxshmfence-1.3' from 'https://cache.nixos.org'...
copying path '/nix/store/nn5l68h9m0dyfmd0wmxhnhgjfbwyrs9p-llvm-7.1.0-lib' from 'https://cache.nixos.org'...
copying path '/nix/store/xcd5wwbsyr38mvrlqfhfgml54fgpa6jh-mirrors-list' from 'https://cache.nixos.org'...
copying path '/nix/store/c7z59k9yv0yyxlrrj1bar5pzg9356k5z-clang-7.1.0' from 'https://cache.nixos.org'...
copying path '/nix/store/hdw7ingz4paq6jq4n341xgfj3xhf9ywv-clang-7.1.0-lib' from 'https://cache.nixos.org'...
copying path '/nix/store/ql5gnzmi02fr566g0wavvls84wffg1j6-llvm-7.1.0' from 'https://cache.nixos.org'...
copying path '/nix/store/vqny9qpqdppmqw51v2jsc72pil3vsii0-nghttp2-1.43.0' from 'https://cache.nixos.org'...
copying path '/nix/store/swi24jhphnkwbwnwlknb7m147s4ix99x-cctools-binutils-darwin-949.0.1' from 'https://cache.nixos.org'...
copying path '/nix/store/hh47i379mm4cbb2dws8gfw712kmdgn9z-nghttp2-1.43.0-lib' from 'https://cache.nixos.org'...
copying path '/nix/store/0mdzl24li08cq8qq8hbp8m1xh03zpa78-cctools-binutils-darwin-wrapper-949.0.1' from 'https://cache.nixos.org'...
copying path '/nix/store/17mkcc40sfzphy9722gy92l0rgnv4z9y-openssl-1.1.1l' from 'https://cache.nixos.org'...
copying path '/nix/store/3v5asy4nb2yqkb3asgvpv6ya7nzh09yh-clang-wrapper-7.1.0' from 'https://cache.nixos.org'...
copying path '/nix/store/sicx2vnd45jk1fn1c14cpwi8qzzs8x54-libssh2-1.9.0' from 'https://cache.nixos.org'...
copying path '/nix/store/2i6iscy7pqkykqp904iplbc7zb2r6z2i-nghttp2-1.43.0-bin' from 'https://cache.nixos.org'...
copying path '/nix/store/vzab3fdy9cvr3sg34mfw0rgbp4p4vj7k-curl-7.76.1' from 'https://cache.nixos.org'...
copying path '/nix/store/czyirh1hh3b9h106iarfj129qv00p3yv-libssh2-1.9.0-dev' from 'https://cache.nixos.org'...
copying path '/nix/store/b5gcf12ij0485gqhbqbnkbisprkl6ij6-curl-7.76.1-bin' from 'https://cache.nixos.org'...
copying path '/nix/store/kc9synymflz7qhxbbdrpqiw0lxwg6cpx-nghttp2-1.43.0-dev' from 'https://cache.nixos.org'...
copying path '/nix/store/9kxlyddw3zgjsrvzxy8hb1nfa1jasgkp-openssl-1.1.1l-bin' from 'https://cache.nixos.org'...
copying path '/nix/store/qh0bysjw5npwdyl6yppbcizlwp2339n5-patch-2.7.6' from 'https://cache.nixos.org'...
copying path '/nix/store/x5cp0jqsg9jz26n3a3vzl53b3wdy71ga-openssl-1.1.1l-dev' from 'https://cache.nixos.org'...
copying path '/nix/store/2dpd685mlhdpdvvnnhshjgbb5n4113cb-stdenv-darwin' from 'https://cache.nixos.org'...
copying path '/nix/store/h5bsb60pqsizr070084v4448bvvgpk76-curl-7.76.1-dev' from 'https://cache.nixos.org'...
copying path '/nix/store/9mlkdg49j77i55v27jdl7kwcza5vvimk-stdenv-darwin' from 'https://cache.nixos.org'...
copying path '/nix/store/4ybj3xpyr0w0h784fjmmz18w9q64y06g-unzip-6.0' from 'https://cache.nixos.org'...
copying path '/nix/store/8vy1z8pbswdishqp62824dbxvgnx9z6z-xorgproto-2021.4' from 'https://cache.nixos.org'...
copying path '/nix/store/wynzvldzsrgv4z4w1sxx8awh2rzdzh1s-libxshmfence-1.3-dev' from 'https://cache.nixos.org'...
building '/nix/store/b1dj9z5qmwf4qkcdfb3mqaw6gqsp6wqn-VSCode_1.60.1_darwin.zip.drv'...

trying https://update.code.visualstudio.com/1.60.1/darwin/stable
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100   118  100   118    0     0    191      0 --:--:-- --:--:-- --:--:--   191
100  102M  100  102M    0     0  41.9M      0  0:00:02  0:00:02 --:--:-- 69.0M
building '/nix/store/ylnzdcy12kdarq9lxbby8bbk4f2fds4j-code-url-handler.desktop.drv'...
Running desktop-file validation
/nix/store/vr39q05h84hw34m0aqcxkr08f2syb3sc-code-url-handler.desktop/share/applications/code-url-handler.desktop: hint: value "Utility;TextEditor;Development;IDE;" for key "Categories" in group "Desktop Entry" contains more than one main category; application might appear more than once in the application menu
building '/nix/store/5bp20nl1yyxsif42y2hzfizk7gsyvh1w-code.desktop.drv'...
Running desktop-file validation
/nix/store/6l3y5n3wjnzdxnhkg9iym999cr99411c-code.desktop/share/applications/code.desktop: hint: value "Utility;TextEditor;Development;IDE;" for key "Categories" in group "Desktop Entry" contains more than one main category; application might appear more than once in the application menu
building '/nix/store/i17ppaa3z36kzcknbmkrg552fda5zffs-vscode-1.60.1.drv'...
unpacking sources
unpacking source archive /nix/store/c7l2vh4071vk0wj4h41k6g1sgs66l06l-VSCode_1.60.1_darwin.zip
source root is Visual Studio Code.app
setting SOURCE_DATE_EPOCH to timestamp 1631319132 of file Visual
patching sources
internal/fs/utils.js:314
    throw err;
    ^

Error: ENOENT: no such file or directory, open 'resources/app/node_modules.asar'
    at Object.openSync (fs.js:498:3)
    at Object.module.exports.readArchiveHeaderSync (/nix/store/ax3zwn40lc1cpkvizj1f2yc54rprs2k2-asar-3.0.3/lib/node_modules/asar/lib/disk.js:58:17)
    at Object.module.exports.readFilesystemSync (/nix/store/ax3zwn40lc1cpkvizj1f2yc54rprs2k2-asar-3.0.3/lib/node_modules/asar/lib/disk.js:84:25)
    at Object.module.exports.extractAll (/nix/store/ax3zwn40lc1cpkvizj1f2yc54rprs2k2-asar-3.0.3/lib/node_modules/asar/lib/asar.js:170:27)
    at Command.<anonymous> (/nix/store/ax3zwn40lc1cpkvizj1f2yc54rprs2k2-asar-3.0.3/lib/node_modules/asar/bin/asar.js:72:10)
    at Command.listener [as _actionHandler] (/nix/store/ax3zwn40lc1cpkvizj1f2yc54rprs2k2-asar-3.0.3/lib/node_modules/asar/node_modules/commander/index.js:413:31)
    at Command._parseCommand (/nix/store/ax3zwn40lc1cpkvizj1f2yc54rprs2k2-asar-3.0.3/lib/node_modules/asar/node_modules/commander/index.js:914:14)
    at Command._dispatchSubcommand (/nix/store/ax3zwn40lc1cpkvizj1f2yc54rprs2k2-asar-3.0.3/lib/node_modules/asar/node_modules/commander/index.js:865:18)
    at Command._parseCommand (/nix/store/ax3zwn40lc1cpkvizj1f2yc54rprs2k2-asar-3.0.3/lib/node_modules/asar/node_modules/commander/index.js:882:12)
    at Command.parse (/nix/store/ax3zwn40lc1cpkvizj1f2yc54rprs2k2-asar-3.0.3/lib/node_modules/asar/node_modules/commander/index.js:717:10) {
  errno: -2,
  syscall: 'open',
  code: 'ENOENT',
  path: 'resources/app/node_modules.asar'
}
error: builder for '/nix/store/i17ppaa3z36kzcknbmkrg552fda5zffs-vscode-1.60.1.drv' failed with exit code 1;
       last 10 log lines:
       >     at Command.listener [as _actionHandler] (/nix/store/ax3zwn40lc1cpkvizj1f2yc54rprs2k2-asar-3.0.3/lib/node_modules/asar/node_modules/commander/index.js:413:31)
       >     at Command._parseCommand (/nix/store/ax3zwn40lc1cpkvizj1f2yc54rprs2k2-asar-3.0.3/lib/node_modules/asar/node_modules/commander/index.js:914:14)
       >     at Command._dispatchSubcommand (/nix/store/ax3zwn40lc1cpkvizj1f2yc54rprs2k2-asar-3.0.3/lib/node_modules/asar/node_modules/commander/index.js:865:18)
       >     at Command._parseCommand (/nix/store/ax3zwn40lc1cpkvizj1f2yc54rprs2k2-asar-3.0.3/lib/node_modules/asar/node_modules/commander/index.js:882:12)
       >     at Command.parse (/nix/store/ax3zwn40lc1cpkvizj1f2yc54rprs2k2-asar-3.0.3/lib/node_modules/asar/node_modules/commander/index.js:717:10) {
       >   errno: -2,
       >   syscall: 'open',
       >   code: 'ENOENT',
       >   path: 'resources/app/node_modules.asar'
       > }
       For full logs, run 'nix log /nix/store/i17ppaa3z36kzcknbmkrg552fda5zffs-vscode-1.60.1.drv'.
```

### Additional context
ran `nix-channel --update` before attempting to install

### Notify maintainers

@eadwu @Synthetica9 @maxeaubrey 

### Metadata
Please run `nix-shell -p nix-info --run "nix-info -m"` and paste the result.

```console
[user@system:~]$ nix-shell -p nix-info --run "nix-info -m"
 - system: `"x86_64-darwin"`
 - host os: `Darwin 20.6.0, macOS 10.16`
 - multi-user?: `yes`
 - sandbox: `no`
 - version: `nix-env (Nix) 2.4pre20210908_3c56f62`
 - channels(user): `"nixpkgs-21.11pre317548.e0ce3c683ae"`
 - channels(root): `""`
 - nixpkgs: `/nix/store/7ag892xlbbr0n07jxc2ipi7m93jqq34j-source`
```

Maintainer information:
```yaml
# a list of nixpkgs attributes affected by the problem
attribute:
# a list of nixos modules affected by the problem
module:
```

**Comment by pseudocubic** [comment]
If I set my channel back to before ecf4ca4c2363373009f875732c832fe8d00f16b0 I am able to successfully install vscode:

`NIXPKGS_ALLOW_UNFREE=1 NIX_PATH='nixpkgs=https://github.com/NixOS/nixpkgs/archive/268c8d77ca7302dd8c72370532e5158a0a3e4a2e.tar.gz' nix-shell -p vscode`

**Comment by prusnak** [comment]
> If I set my channel back to before [ecf4ca4](https://github.com/NixOS/nixpkgs/commit/ecf4ca4c2363373009f875732c832fe8d00f16b0) I am able to successfully install vscode:

This commit comes from https://github.com/NixOS/nixpkgs/pull/137912  // cc @pshirshov @SuperSandro2000

**Comment by Artturin** [comment]
maybe something to do with darwin?

**Comment by SuperSandro2000** [comment]
I didn't even look at that yet. Builds fine on Linux, for Darwin I no longer want to invest time. If you find a solution feel free to submit a PR. Maybe it is already enough to gate this part of the code behind a ``lib.optionalString stdenv.isLinux``?

**Comment by happysalada** [comment]
Just wondering if that new PR fixes your bug https://github.com/NixOS/nixpkgs/pull/139041

**Comment by onethirtyfive** [comment]
> Just wondering if that new PR fixes your bug #139041

I've been following this because I ran into the same failure. Unfortunately, no dice. Given this expression, part of my dotfiles:

```
let
  pinnedPkgs = import (builtins.fetchGit {
    name = "custom-vscode-1.60.2";
    url = "https://github.com/samuela/nixpkgs/";
    ref = "refs/heads/upkeep-bot/vscode-1.60.2-1632356161";
    rev = "a946fb970f985e20d038e9d12c0db68a8b3b2f19";
  }) {};

  extensions = (with pinnedPkgs.vscode-extensions; [
    bbenoist.nix
    vscodevim.vim
  ] ++ pkgs.vscode-utils.extensionsFromVscodeMarketplace [{
      name = "nix-env-selector";
      publisher = "arrterian";
      version = "1.0.7";
      sha256 = "0mralimyzhyp4x9q98x3ck64ifbjqdp8cxcami7clvdvkmf8hxhf";
  }]);
in pinnedPkgs.vscode-with-extensions.override {
  vscodeExtensions = extensions;
}
```

I get the same build failure:

```
building '/nix/store/x9j6la0656ly0s3gvigaak9281s5acny-vscode-1.60.2.drv'...
unpacking sources
unpacking source archive /nix/store/99a06l7vgp4qi4yxsspd4m1zr6wf313y-VSCode_1.60.2_darwin-arm64.zip
source root is Visual Studio Code.app
setting SOURCE_DATE_EPOCH to timestamp 1632314212 of file Visual
patching sources
internal/fs/utils.js:314
    throw err;
    ^

Error: ENOENT: no such file or directory, open 'resources/app/node_modules.asar'
    at Object.openSync (fs.js:498:3)
    at Object.module.exports.readArchiveHeaderSync (/nix/store/f7jhdvdaxf7xlbilgrybkni2mahp95b9-asar-3.1.0/lib/node_modules/asar/lib/disk.js:58:17)
    at Object.module.exports.readFilesystemSync (/nix/store/f7jhdvdaxf7xlbilgrybkni2mahp95b9-asar-3.1.0/lib/node_modules/asar/lib/disk.js:84:25)
    at Object.module.exports.extractAll (/nix/store/f7jhdvdaxf7xlbilgrybkni2mahp95b9-asar-3.1.0/lib/node_modules/asar/lib/asar.js:174:27)
    at Command.<anonymous> (/nix/store/f7jhdvdaxf7xlbilgrybkni2mahp95b9-asar-3.1.0/lib/node_modules/asar/bin/asar.js:72:10)
    at Command.listener [as _actionHandler] (/nix/store/f7jhdvdaxf7xlbilgrybkni2mahp95b9-asar-3.1.0/lib/node_modules/asar/node_modules/commander/index.js:413:31)
    at Command._parseCommand (/nix/store/f7jhdvdaxf7xlbilgrybkni2mahp95b9-asar-3.1.0/lib/node_modules/asar/node_modules/commander/index.js:914:14)
    at Command._dispatchSubcommand (/nix/store/f7jhdvdaxf7xlbilgrybkni2mahp95b9-asar-3.1.0/lib/node_modules/asar/node_modules/commander/index.js:865:18)
    at Command._parseCommand (/nix/store/f7jhdvdaxf7xlbilgrybkni2mahp95b9-asar-3.1.0/lib/node_modules/asar/node_modules/commander/index.js:882:12)
    at Command.parse (/nix/store/f7jhdvdaxf7xlbilgrybkni2mahp95b9-asar-3.1.0/lib/node_modules/asar/node_modules/commander/index.js:717:10) {
  errno: -2,
  syscall: 'open',
  code: 'ENOENT',
  path: 'resources/app/node_modules.asar'
}
builder for '/nix/store/x9j6la0656ly0s3gvigaak9281s5acny-vscode-1.60.2.drv' failed with exit code 1
cannot build derivation '/nix/store/igf3wbsnpfqgqaxdx1r01filw8sl10s4-vscode-with-extensions-1.60.2.drv': 1 dependencies couldn't be built
```

System context:

```
$ nix-shell -p nix-info --run "nix-info -m"
 - system: `"aarch64-darwin"`
 - host os: `Darwin 20.6.0, macOS 11.6`
 - multi-user?: `no`
 - sandbox: `no`
 - version: `nix-env (Nix) 2.3.15`
 - channels(onethirtyfive): `"nixpkgs-21.11pre317242.bc9b956714e"`
 - nixpkgs: `/Users/onethirtyfive/.nix-defexpr/channels/nixpkgs`
```

**Comment by Artturin** [comment]
The postpartum should be made linux only
